juju-tutorials/how-to-deploy-openstack-on-openstack.md

```diff
@@ -226,16 +226,17 @@ Don't forget to get credentials for the new OpenStack. (e.g. `source openrc`, wh
       ```
       [note="credentials example]
       ```yaml
-      <cloud_name>:
-        <credentials name>:
-          auth-type: userpass
-          username: admin
-          password: <user_password>
-          domain-name: ""
-          project-domain-name: admin_domain
-          tenant-name: admin
-          user-domain-name: admin_domain
-          version: "3"
+      credentials:
+        <cloud_name>:
+          <credentials name>:
+            auth-type: userpass
+            username: admin
+            password: <user_password>
+            domain-name: ""
```

Comment:
I think the `domain-name: ""` definition is not needed. I think Juju would set it to `null`, which has the same effect.

Other than this, the change looks good.